ui/src/components/Document/DocumentLanguages.js

```diff
@@ -1,40 +1,53 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { List } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
-import isEmpty from 'lodash/isEmpty';
+import _get from 'lodash/get';
 
-export class DocumentLanguages extends Component {
-  render() {
-    const { metadata, delimiter } = this.props;
-    return (
-      <>
-        {this.props.prefix}
-        <List horizontal className={'document-languages-list'}>
-          {metadata && !isEmpty(metadata.languages)
-            ? metadata.languages.map((language, index) => (
-                <List.Item
-                  as={this.props.listItemAs ? this.props.listItemAs : ''}
-                  key={`Key${index}`}
-                >
-                  {language.toUpperCase()}
-                  {index !== metadata.languages.length - 1 ? delimiter : null}
-                </List.Item>
-              ))
-            : null}
-        </List>
-      </>
-    );
+export const DocumentLanguages = ({
+  metadata,
+  separator,
+  withLabel,
+  listItemAs,
+}) => {
+  const languages = _get(metadata, 'languages', []);
+  if (!languages.length) {
```

Comment:
`isEmpty` like everywhere else, instead of length comparison?

Reply:
honestly, where we can use plain javascript, let's go with vanilla! :)